sdk/packages/core/src/api/getQuote.ts

```diff
@@ -1,10 +1,23 @@
-import { type Address, type Hex, fromHex, zeroAddress } from 'viem'
+import { string, z } from 'zod'
+import { fromHex, Hex, zeroAddress } from 'viem'
 import { fetchJSON } from '../internal/api.js'
 import type { Environment } from '../types/config.js'
 import type { Quote, Quoteable } from '../types/quote.js'
 import type { Prettify } from '../types/utils.js'
 import { getApiUrl } from '../utils/getApiUrl.js'
 import { toJSON } from '../utils/toJSON.js'
+import { address, hex } from '../schema/types.js'
+
+export const quoteResponseSchema = z.object({
+  deposit: z.object({
+    token: address(),
+    amount: z.union([hex(), string()]),
+  }),
+  expense: z.object({
+    token: address(),
+    amount: z.union([hex(), string()]),
+  }),
+})
```

Comment:
`amount` could be string in test data, so i made `amount` to `z.union([hex(), string()])`